src/docfx/build/redirection/RedirectionMap.cs

```diff
@@ -79,31 +73,25 @@ void AddRedirections(Dictionary<string, string> items, bool checkRedirectTo = fa
                         continue;
                     }
 
-                    if (checkRedirectTo && !redirectTo.StartsWith('/'))
+                    if (checkRedirectTo && !redirectTo.Value.StartsWith('/'))
```

Comment:
could `redirectTo` be null?

Reply:
I think it shouldn't be `null`, otherwise that'll complicate everything (see existing references to `SourceInfo<T>.Value`), I added a TODO to eliminate null problems associated with `SourceInfo`